corehq/apps/app_manager/static/app_manager/js/releases.js

```diff
@@ -253,12 +253,14 @@ hqDefine('app_manager/js/releases.js', function () {
             var lastApp = self.savedApps()[0];
             self.lastAppVersion(lastApp ? lastApp.version() : -1);
         });
-        self.url = function (name) {
-            var template = self.options.urls[name];
+        self.reverse = function() {
             for (var i = 1; i < arguments.length; i++) {
-                template = template.replace('___', ko.utils.unwrapObservable(arguments[i]));
+                arguments[i] = ko.utils.unwrapObservable(arguments[i]);
```

Comment:
It does seem kind of annoying. I think I'd still end up having a function here, though, if only because of the places where `reverse` is called from within a `data-bind`, it seems clunky to do an entire `hqImport(...).reverse(...)` in html. So I'm inclined to leave it alone.